src/redirect.rs

```diff
@@ -0,0 +1,204 @@
+//! See [`Redirect`] for service/responder documentation.
+
+use std::{borrow::Cow, ops::Deref};
+
+use actix_service::fn_service;
+use futures_util::future::ready;
+
+use crate::{
+    dev::{AppService, HttpServiceFactory, ResourceDef, ServiceRequest},
+    http::{header, StatusCode},
+    HttpRequest, HttpResponse, Responder,
+};
+
+/// An HTTP service for redirecting one path to another path or URL.
+///
+/// Redirects are either [relative](Redirect::to) or [absolute](Redirect::to).
+///
+/// By default, the "308 Temporary Redirect" status is used when responding.
+/// See [this MDN article](mdn-redirects) on why 308 is preferred over 301.
```

Comment:
308 is the permanent redirect.
Link goes to temporary redirect section, but 308 is listed in the permanent redirect section above it.